config/ingest.go

```diff
@@ -36,16 +39,23 @@ type Ingest struct {
 
 	// PublisherKind specifies which legs.Publisher implementation to use.
 	PublisherKind PublisherKind
+
+	// StartPublishDelay specifies how long to wait, after startup, to
+	// re-publish the latest advertisement notification.  This delay gives time
+	// to connect to other nodes and establish gossip mesh before sending
+	// pubsub advertisement notification message.
+	StartPublishDelay Duration
```

Comment:
`PublishLatestAdOnStartDelay`? Or a name that makes it clear this only applies to republishing of the latest ad?

Reply:
I think I would rather change this to:
```go
// AutoAnnounceInterval is the amount of time to elapse, in which no new announcements
// have been published, until the latest advertisement CID announcement is republished. 
AutoAnnounceInterval Duration
```
and default it to 12 hours, and have a command `provider announce` to announce the latest ad CID on demand.